spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -633,7 +639,12 @@ def __init__(self, file_path, electrical_series=None, sampling_frequency=None):
                 else:  # if it is unit_property
                     for id in units_ids:
                         ind = list(units_ids).index(id)
-                        self.set_unit_property(id, item, nwbfile.units[item][ind])
+                        if isinstance(nwbfile.units[item][ind], pd.DataFrame):
```

Comment:
@alejoe91 in what situation would `nwbfile.units[item][ind]` be a `pd.DataFrame`? Is this something you added for tests?

Reply:
@CodyCBakerPhD added a `table` arg to the [`set_dynamic_table_property ` ](https://github.com/SpikeInterface/spikeextractors/pull/430/files#diff-35851cf23629fc4a9bddde08ed99b1c2R31).

When the sorting have `max_channel` or `max_electrode` properties, the [`nwbfile.electrodes`](https://github.com/SpikeInterface/spikeextractors/pull/430/files#diff-35851cf23629fc4a9bddde08ed99b1c2R807) are used as `table` for the `add_column` function. 

When retrieving those properties, they are returned as a pandas df with electrode id as index